Dockerfile

```diff
@@ -68,38 +76,38 @@ RUN set -x \
     && grep " node-v$NODE_VERSION-linux-x64.tar.gz\$" SHASUMS256.txt.asc | sha256sum -c - \
     && tar -xzf "node-v$NODE_VERSION-linux-x64.tar.gz" -C /usr/local --strip-components=1 \
     && rm -r "$GNUPGHOME" "node-v$NODE_VERSION-linux-x64.tar.gz" SHASUMS256.txt.asc \
-    && apt-get purge -y --auto-remove wget
-
-RUN npm install -g yarn
+    && apt-get purge -y --auto-remove wget \
+    && npm install -g yarn@$YARN_VERSION \
+    && npm cache clear --force
 
-COPY yarn.lock /usr/src/app/
-COPY package.json /usr/src/app/
-RUN yarn install
+COPY package.json yarn.lock /usr/src/zeus/
+RUN yarn install --production --pure-lockfile --ignore-optional \
+    && yarn cache clean
 
-COPY requirements-base.txt /usr/src/app/
+COPY requirements-base.txt /usr/src/zeus/
 RUN pip install -r requirements-base.txt
 
-COPY requirements-dev.txt /usr/src/app/
+COPY requirements-dev.txt /usr/src/zeus/
 RUN pip install -r requirements-dev.txt
 
-COPY requirements-test.txt /usr/src/app/
+COPY requirements-test.txt /usr/src/zeus/
 RUN pip install -r requirements-test.txt
 
-COPY . /usr/src/app
-RUN pip install -e .
-RUN node_modules/.bin/webpack --config=config/webpack.config.dev.js
+COPY . /usr/src/zeus
+RUN pip install -e . \
+    && NODE_ENV=production node_modules/.bin/webpack --config=config/webpack.config.dev.js
 
 ENV REPO_ROOT /workspace/repos
 RUN mkdir -p $REPO_ROOT
 
-ENV PATH /usr/src/app/bin:$PATH
+ENV PATH /usr/src/zeus/bin:$PATH
 
-# Make port 8080 available to the world outside this container
+# Make port 5000 available to the world outside this container
 EXPOSE 5000
 
 VOLUME /workspace
 
-ENTRYPOINT ["/usr/src/app/bin/docker-entrypoint"]
+ENTRYPOINT ["docker-entrypoint"]
 
 # Run Zeus
 CMD ["zeus", "run", "--host=0.0.0.0", "--port=5000"]
```

Comment:
Also, I'd recommend letting zeus read an optional `PORT` environment variable instead of hardcoding 5000 here. That way it's possible to run without port mapping in environments that allow that. Not sure if Kubernetes gives you this option or not. But usually a scheduler will pass `PORT=xxxx` and you can then bind correctly with host networking instead of port forwarding.